src/features/homepage/Details.js

```diff
@@ -0,0 +1,109 @@
+import { Link, useLocation } from 'react-router-dom';
+import {
+  BsMic, BsGear, BsChevronLeft, BsArrowRightCircle,
+} from 'react-icons/bs';
+import { Img } from './Card';
+import './css/Homepage.css';
+import './css/Details.css';
+
+const Details = () => {
+  const location = useLocation();
+  const quote = location.state;
+  return (
+    <main className="details">
+      <nav id="details-nav">
+        <Link to="/"><BsChevronLeft className="navIcon" /></Link>
```

Comment:
- [ ] I'm sure this arrow is meant to take the user back from the details page, but I noticed that the App.js routes you set for the homepage do not match this one for the back arrow. Currently, when the back arrow is clicked, the page routes to an empty 404 page. Kindly fix the link to the homepage from the details page so the app works fine locally and the hosted one as well. 👍🏽